python/cudf/cudf/tests/test_parquet.py

```diff
@@ -806,3 +806,23 @@ def test_write_cudf_read_pandas_pyarrow(tmpdir, pdf):
     ).to_pandas()
 
     assert_eq(cudf_res, pd_res, check_index_type=False if pdf.empty else True)
+
+
+def test_parquet_writer_criteo(tmpdir):
+    # To run this test, download the day 0 of criteo dataset from
+    # http://labs.criteo.com/2013/12/download-terabyte-click-logs/
+    # and place the uncompressed dataset in the home directory
+    fname = os.path.expanduser("~/day_0")
+    if not os.path.isfile(fname):
```

Comment:
Related: `os.path.isfile` fails for paths without tilde expansion. This other test doesn't seem to expand the path first so it seems it would always be skipped https://github.com/rapidsai/cudf/blob/542e4e4a66e420bb63f0ba317ac47b50645f3402/python/cudf/cudf/tests/test_parquet.py#L467-L469

Reply:
Can you fix it in this PR? Nice catch